code/modules/ninja/suit/n_suit_verbs/ninja_net.dm

```diff
@@ -1 +1,33 @@
-// TODO
\ No newline at end of file
+
+//Allows the ninja to kidnap people
+/obj/item/clothing/suit/space/space_ninja/proc/ninjanet()
+	var/mob/living/carbon/human/H = affecting
+	var/mob/living/carbon/C = input("Select who to capture:","Capture who?",null) as null|mob in oview(H)
+
+	if(QDELETED(C)||!(C in oview(H)))
+		return 0
+
+	if(!C.client)//Monkeys without a client can still step_to() and bypass the net. Also, netting inactive people is lame.
```

Comment:
This means going SSD is a defense against ninjanets (do the ninjas have any objectives to bring people back to base?)

Reply:
They have capture objectives. This is to prevent cheese like farming monkies for the objective (or humanized monkies). Similar to rev, shadowling, cult, etc.